include/oqpi/threading/interface_thread.hpp

```diff
@@ -3,6 +3,7 @@
 #include <tuple>
 #include <memory>
 #include <type_traits>
+#include <functional>
```

Comment:
What do we need in functional?

Reply:
because of std::invoke! it wont build with g++ or clang++ without including functional